recipes/aruco/3.x.x/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class LibnameConan(ConanFile):
+    name = "aruco"
+    description = "Augmented reality library based on OpenCV "
+    topics = ("conan", "aruco", "augmented reality")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.uco.es/investiga/grupos/ava/node/26"
+    license = "GPL-3.0-only"
+    no_copy_source = True
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [False, True], "fPIC": [False, True]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("opencv/4.5.0@")
+        self.requires("eigen/3.3.8@")
+        self.requires("zlib/1.2.11@")
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version], filename="aruco.zip"
```

Comment:
usually we do not need filename, might be apart of madebr's suggestion for URL

Reply:
After @madebr suggestions, this is indeed not necessary anymore.